internal/services/synapse/synapse_sql_pool_resource.go

```diff
@@ -337,6 +331,7 @@ func resourceSynapseSqlPoolRead(d *pluginsdk.ResourceData, meta interface{}) err
 			d.SetId("")
 			return nil
 		}
+
 		return fmt.Errorf("retrieving Synapse SqlPool %q (Workspace %q / Resource Group %q): %+v", id.Name, id.WorkspaceName, id.ResourceGroup, err)
```

Comment:
Looks like this was one, the log message in [line 330](https://github.com/hashicorp/terraform-provider-azurerm/blob/b8312c1e64ef47d428814430d40d80d93aec71f4/internal/services/synapse/synapse_sql_pool_resource.go#L330) and [line 340](https://github.com/hashicorp/terraform-provider-azurerm/blob/b8312c1e64ef47d428814430d40d80d93aec71f4/internal/services/synapse/synapse_sql_pool_resource.go#L340) was forgotten
```
return fmt.Errorf("retrieving %s: %+v", id, err)
```